docs/README.md

```diff
@@ -229,6 +229,14 @@ Here's an example configuration file for advanced settings and all of it's possi
         must_exclude:
           regex: 'DO NOT MERGE'
           message: 'Custom message...'
+          
+      files:
+        # Check each modified file in the PR for the following starting at line 0.
+        header: > 
+          Entire license header.
+        # only enforce it for these file patterns uses the same rules as CODEOWNER file
```

Comment:
The `CODEOWNERS` file follows the same file matching patterns as the `gitignore`. I suggest having this link (https://git-scm.com/docs/gitignore#_pattern_format) in the comment above to be clear.

Reply:
done